packages/p2p-v2/src/components/PaymentMethodCard/PaymentMethodCard.tsx

```diff
@@ -1,41 +1,85 @@
 import React, { HTMLAttributes } from 'react';
-import { TAdvertiserPaymentMethods } from 'types';
+import clsx from 'clsx';
+import { TPaymentMethod } from 'types';
+import { LabelPairedPlusLgBoldIcon } from '@deriv/quill-icons';
+import { Button, Text } from '@deriv-com/ui';
 import { PaymentMethodCardBody } from './PaymentMethodCardBody';
 import { PaymentMethodCardHeader } from './PaymentMethodCardHeader';
 import './PaymentMethodCard.scss';
 
 type TPaymentMethodCardProps = HTMLAttributes<HTMLDivElement> & {
     isEditable?: boolean;
-    isSelectable?: boolean;
-    onDeletePaymentMethod: () => void;
-    onEditPaymentMethod: () => void;
-    paymentMethod: NonNullable<TAdvertiserPaymentMethods>[number];
+    medium?: boolean;
+    onClickAdd?: (paymentMethod: TPaymentMethod) => void;
+    onDeletePaymentMethod?: () => void;
+    onEditPaymentMethod?: () => void;
+    onSelectPaymentMethodCard?: (paymentMethodId: number) => void;
+    paymentMethod: TPaymentMethod & { isAvailable?: boolean };
+    selectedPaymentMethodIds?: number[];
     shouldShowPaymentMethodDisplayName?: boolean;
 };
 
 const PaymentMethodCard = ({
     isEditable = false,
-    isSelectable = false,
-    onDeletePaymentMethod,
-    onEditPaymentMethod,
+    medium = false,
+    onClickAdd = () => {
+        //Do nothing
+    },
+    onDeletePaymentMethod = () => {
+        //Do nothing
+    },
+    onEditPaymentMethod = () => {
+        //Do nothing
+    },
+    onSelectPaymentMethodCard = () => {
+        //Do nothing
+    },
```

Comment:
do we need to define it as an empty function here?

Reply:
would be better to define a function rather than leaving it undefined.